library/Icingadb/Model/Host.php

```diff
@@ -138,6 +145,27 @@ public function createBehaviors(Behaviors $behaviors)
         ]));
     }
 
+    /**
+     * Mutates flattened custom vars to an associative array
+     *
+     * @param  Query|ResultSet $queryOrResult
+     *
+     * @return array
+     */
+    public function mutateVarsProperty($queryOrResult): array
+    {
+        if (! $queryOrResult instanceof ResultSet) {
```

Comment:
I noticed that this receives always a query. Remember this?:

> * In the Host and Service models, add a new method mutateVarsProperty()
>   * This transforms customvar_flat to an associative array where cv names are keys and cv values the values.
>   * If customvar_flat is not a ResultSet, apply restrictions to it (the query) beforehand

You have to access `$this->customvar_flat` instead of `$queryOrResult`.